tests/test_result_to_dict.py

```diff
@@ -200,6 +203,16 @@ def test_callback(self):
         data = result_to_dict(json_value_without_str_obj, callback=convert_return_none)
         self.assertDictEqual(data, json_value_without_str_obj)
 
-    @ResultToDict()

```

Comment:
why was this function removed? we are using this function to test the decorator and is being used in the test cases, this will cause the test to fail if we remove it,
can you please add this back, thank you.